crates/common/tedge_config_macros/impl/src/reader.rs

```diff
@@ -67,12 +71,38 @@ fn generate_structs(
                     false => parse_quote!(pub),
                 });
             }
+            // TODO skipping?
+            FieldOrGroup::Multi(group) => {
+                // TODO is this right?
+                let sub_reader_name = prefixed_type_name(name, group);
+                idents.push(&group.ident);
+                tys.push(parse_quote_spanned!(group.ident.span()=> ::tedge_config_macros::Multi<#sub_reader_name>));
+                let mut parents = parents.clone();
+                parents.push(PathItem::Static(group.ident.clone()));
```

Comment:
I should remove `PathItem` as it's a less flexible alternative to what we are using `IdGenerator` for earlier on.

Reply:
Didn't remove it, but have used `SequentialIdGenerator` to do the ID generation.